tracpro/polls/tasks.py

```diff
@@ -92,7 +92,9 @@ def fetch_org_updated_runs(org):
     incomplete_responses = Response.get_update_required(org)
 
     if incomplete_responses:
-        runs = client.get_runs(ids=[r.flow_run_id for r in incomplete_responses])
+        runs = []
+        for i in range(0, len(incomplete_responses), 15):
```

Comment:
But is that not a bit risky? Say if we have 16 responses every 5
minute(frequency of the task), then we will have about 12 delayed every
hour. I am not sure how that will impact the system but I would prefer not
to find out. What would be the impact of trying to get them all in one task?

On Thu, Apr 16, 2015 at 9:45 AM, Evan Wheeler <notifications@github.com>
wrote:

> In tracpro/polls/tasks.py
> <https://github.com/rapidpro/tracpro/pull/6#discussion_r28487225>:
>
> > @@ -92,7 +92,9 @@ def fetch_org_updated_runs(org):
> >      incomplete_responses = Response.get_update_required(org)
> >
> >      if incomplete_responses:
> > -        runs = client.get_runs(ids=[r.flow_run_id for r in incomplete_responses])
> > +        runs = []
> > +        for i in range(0, len(incomplete_responses), 15):
>
> @xkmato <https://github.com/xkmato> true, but this is a task that is
> repeated regularly. so with @nicpottier <https://github.com/nicpottier>'s
> version only 15 are synced each time the task is run. will eventually get
> them all
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rapidpro/tracpro/pull/6/files#r28487225>.
>